test/npb/mg.chpl

```diff
@@ -48,7 +48,7 @@ config const warmup = true;
 type coeff = [0..3] real;
 
 const Levels: domain(1) = {1..numLevels};
-const Base: domain(3) dmapped(Block(3)) = {1..nx, 1..ny, 1..nz};
```

Comment:
These also flag as weird to me that its `dmapped(blockDist)` instead of just `dmapped blockDist`

Reply:
This isn't actually tested and is arguably in the repo for archival purposes.  Arguably I shouldn't have changed it, but it was easier/more mindless to do a blanket renaming than to determine which tests were run vs. not.